### ISSUE TITLE
Can you provide the pre-trained weights?

### ISSUE THREAD

**Issue by mxtsecure** [xinleihe/toxic-prompt#1]

**Comment by xinleihe** [comment]
Yes, the pre-trained weights for task 1 will be provided later accordingly.

**Comment by mxtsecure** [comment]
Thanks! What about the Weights for task 3?

**Comment by xinleihe** [comment]
Hi, the model weights for task 1 are online now. Currently, we do not save the model weights for tasks 2 and 3 as the training procedure won't take a long time.

**Comment by pkulium** [comment]
@xinleihe 
Simple question: can we get code to plot figures? such as figure 1. Your figure format is pretty nice!
